examples/unimp/unimp_trainer.py

```diff
@@ -0,0 +1,168 @@
+import os
+os.environ['CUDA_VISIBLE_DEVICES']='0'
+import random
+import argparse
+import tensorlayerx as tlx
+import tensorlayerx.nn as nn
+from gammagl.utils import segment_softmax
+from gammagl.datasets import Planetoid
+from gammagl.layers.conv import MessagePassing
+from gammagl.utils import add_self_loops, mask_to_index
+from tensorlayerx.model import TrainOneStep, WithLoss
+
+class CrossEntropyLoss(WithLoss):
+    def __init__(self, model, loss_func):
+        super(CrossEntropyLoss, self).__init__(model,loss_func)
+
+    def forward(self, data, label):
+        out = self.backbone_network(data['x'], data['edge_index'])
+        out = tlx.gather(out, data['val_idx'])
+        label = tlx.reshape(tlx.gather(label, data['val_idx']),shape=(-1,))
+        #print(out[0])
+        #print(label[0])
+        loss = self._loss_fn(out, label)
+        return loss
+
+
+class MultiHead(MessagePassing):
```

Comment:
Put this section into ``gammagl.layers.conv.unimp_conv.py``. If you put this into this file, users will not be able to use this function.